### ISSUE TITLE
UI does not scale well on high resolution monitors

### ISSUE THREAD

**Issue by holly-hacker** [ConcreteMC/Alex#82]
The default UI scaling is set to 1, which is very small on a 4k display. I propose changing it to 2 by default.

Here's comparisons on both 4k and 1080p:

| UI Scaling 1 on 4k                                 | UI Scaling 2 on 4k                                 |
| -------------------------------------------------- | -------------------------------------------------- |
| ![4k, scaling 1](https://i.imgur.com/6wWEiz3.jpeg) | ![4k, scaling 2](https://i.imgur.com/bcGqMRJ.jpeg) |

| UI Scaling 1 on 1080p                                | UI Scaling 2 on 1080p                                |
| ---------------------------------------------------- | ---------------------------------------------------- |
| ![1080p, scaling 1](https://i.imgur.com/4uccs8m.png) | ![1080p, scaling 2](https://i.imgur.com/uv0XLme.png) |

**Comment by kennyvv** [comment]
I agree that the default scaling doesn't feel right anyways, maybe we should look into changing the way it does this.
